data/oil/NO/NO00003.json

```diff
@@ -215,7 +226,7 @@
                             "unit_type": "volumefraction"
                         },
                         "vapor_temp": {
-                            "value": 754.8000000000001,
+                            "value": 754.7999999999998,
```

Comment:
This is an awful lot of digits -- probably a result of floating point calculations. It won't do any harm, so I"m going to merge, but it would be nice to clean up some day.

Reply:
These extra decimals are added in the final step, when exporting an Adios oil object to file with `o.to_file()`
https://github.com/NOAA-ORR-ERD/adios_oil_database/blob/c3d87d7a9e70717211dfe1850d807586ef0cbdd2/adios_db/adios_db/models/oil/oil.py#L158
https://github.com/nlohmann/json/issues/1109

I believe this can be solved by adding an "encoder" to `json.dump`:
https://stackoverflow.com/questions/1447287/format-floats-with-standard-json-module